### PR TITLE
Added use volume keys to digitize

### PR DESCRIPTION
**Pull request by SeqLaz**
Added use volume keys to digitize in Digitizing section

### CODE REVIEW

**Comment by duke-nyuki** [comment]
Task linked: [QF-3215 #4594 Implement digitizing throuhg volume keys on Android devices](https://app.clickup.com/t/2192114/QF-3215)

File: documentation/how-to/digitize.en.md [hunk 1]
```diff
@@ -48,6 +48,13 @@ While digitizing, you can click the *(x) button* to cancel the current feature c
     QField insures that digitized geometries will not have duplicate vertices and respects
     the geometry precision settings from the currently selected layer.
 
+You have the option to enable or disable the functionality that allows you to use the volume keys for adding or removing vertices while in digitizing mode. You can activate this functionality in Settings in the General tab.
+
+!![](../assets/images/activate-digitize-with-volumen-keys.png)
+
+!!! note
+    This feature is available on Android.
```
Comment (suricactus) [inline documentation/how-to/digitize.en.md:56]:
```suggestion
    This feature is available on Android only.
```

File: documentation/how-to/digitize.en.md [hunk 2]
```diff
@@ -48,6 +48,13 @@ While digitizing, you can click the *(x) button* to cancel the current feature c
     QField insures that digitized geometries will not have duplicate vertices and respects
     the geometry precision settings from the currently selected layer.
 
+You have the option to enable or disable the functionality that allows you to use the volume keys for adding or removing vertices while in digitizing mode. You can activate this functionality in Settings in the General tab.
```
Comment (suricactus) [inline documentation/how-to/digitize.en.md:51]:
```suggestion
You can use the volume keys for adding or removing vertices while in digitizing mode. This functionality can be activated in Settings in the General tab.
```

Too much suspense before we get to the point.

**Review by suricactus — APPROVED** [review]
Thanks a lot, looks good for me. Two small wording comments.

**Review by suricactus — APPROVED** [review]
Thanks a lot, looks good for me. Two small wording comments.

**Comment by SeqLaz** [comment]
> Thanks a lot, looks good for me. Two small wording comments.

Great improvments

**Pull request merged**